backend/api/Database/Models/MissionTask.cs

```diff
@@ -210,9 +199,15 @@ public static string CalculateHashFromTasks(IList<MissionTask> tasks)
                 var taskCopy = new MissionTask(task)
                 {
                     Id = "",
-                    IsarTaskId = ""
+                    IsarTaskId = "",
                 };
-                taskCopy.Inspections = taskCopy.Inspections.Select(i => new Inspection(i, useEmptyIDs: true)).ToList();
+                if (taskCopy.Inspection is not null)
+                {
+                    taskCopy.Inspection = new Inspection(taskCopy.Inspection, useEmptyIDs: true)
+                    {
+                        IsarTaskId = ""
```

Comment:
Why did we remove this functionality from the use of the useEmptyIDs flag in the constructor if we readd that functionality here?